src/button.styles.ts

```diff
@@ -168,12 +170,24 @@ export default [
       }
     }
 
+    .tooltip {
+      max-inline-size: 100%;
+      white-space: inherit;
+    }
+
     .prefix-icon-slot {
       &.hidden {
         display: none;
       }
     }
 
+    .label {
+      min-inline-size: 3ch;
+      overflow: hidden;
```

Comment:
You'll notice some changes in the visual test report. I [tracked it down](https://github.com/CrowdStrike/glide-core/pull/1057/files#diff-fa785b7755e9fdaf8c5323b6a86b1066de0d47b94dca15d79e7980ea0dd56f0dR185) to this line. For some reason, adding `overflow: hidden` causes a slight visual shift in CI, but not in a real browser. 🤔 

Font smoothing/antialiasing thing in Linux maybe? That's about all I can think of.